content/docs/studio/how-it-works.md

```diff
@@ -35,8 +35,8 @@ https://www.youtube.com/watch?v=5xM5az78Lrg
   [add the metrics and hyperparameters to your Git repositories manually](/doc/studio/user-guide/projects-and-experiments/configure-a-project#custom-metrics-and-parameters).
 - Details of your ML models, including versions and stage assignments, are
   extracted from the Git commits and tags.
-- Live metrics and plots for experiments committed to Git are extracted from the
-  data you send using [DVCLive].
+- Live metrics and plots for experiments are extracted from the data you send
+  using [DVCLive].
```

Comment:
This seems to go in the opposite direction of what https://github.com/iterative/dvc.org/issues/4279 requires. Is it just a loosely related correction?

> p.s. also relates to https://github.com/iterative/dvc.org/pull/4285#pullrequestreview-1281511079

Reply:
How is this opposite?

Reply:
Req:

> the baseline commit ... must be pushed

This:

> ~~committed to Git~~

Just seems to be either unrelated or contradictory.

Reply:
The experiment **for which we are sending live metrics** does not need to be pushed to Git. Only **its parent** needs to be pushed to Git.